src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -50,26 +41,34 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
             }
 
             if (Date.now() > detectorTimeoutMs) {
-                if (showOutput) {
-                    const noIssuesFound: string = `Diagnosing for "${node.root.client.siteName}" has timed out.`;
-                    ext.outputChannel.appendLog(noIssuesFound);
-                    context.telemetry.properties.timedOut = 'true';
-                }
+                context.telemetry.properties.timedOut = 'true';
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: number = Date.now();
+            const startTimeDate: Date = new Date(nowTime - (60 * 60 * 1000));
```

Comment:
Okay I think I understand this now - is this right?

> There's a requirement (outside the control of the LinuxLogDetector team) that these times have to be less than the current time by 14.5 minutes. That being said, the LinuxLogDetector team ignores the times and always gets the most recent entry. We'll use 30/60 minutes just to make sure we're well over 14.5

If so, can we make that the comment in the code? The most important piece is I want the comment to explicitly reference 30/60 otherwise I feel like it's really confusing